addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2242,6 +2242,9 @@ export class OdooEditor extends EventTarget {
      * @param {Node} node
      */
     _cleanForPaste(node) {
+        if (node.nodeName === 'FONT') {
+            return;
+        }
```

Comment:
I'm very confused. `FONT` tags are already allowed. Why is this needed ? And with this return here, why do we still need to add more code below if the function returns directly for `FONT` tags ?

Reply:
Hello @dmo-odoo 
when we copy-paste from discord. the copied text that we paste in the editor comes in a span tag with color style instead of a font tag.
so in my changes, I have converted that span tag into a font tag.

if I don't return the font tag here then it will be going into an infinite loop because of the else if part from where _cleanForPaste calls again.

Reply:
I don't think this is the right fix. I think it would be better to make a change in the function that actually sets the background color to make it recognize span tags as well as font tags.

Reply:
Done.